vnext/local-cli/generator-windows/templates/cpp/src/MainPage.cpp

```diff
@@ -0,0 +1,46 @@
+﻿#include "pch.h"
+#include "MainPage.h"
+#include "MainPage.g.cpp"
+
+using namespace winrt;
+using namespace Windows::UI::Xaml;
+using namespace react::uwp;
+
+// clang-format off
+
+namespace winrt::<%=ns%>::implementation
+{
+	const wchar_t* JSFILENAME = L"index";
+	const wchar_t* JSCOMPONENTNAME = L"<%=name%>";
+
+    MainPage::MainPage()
+    {
+        InitializeComponent();
+		LoadReact();
```

Comment:
>		 [](start = 0, length = 2)

nit: please use space. it applies to other .h and .cpp files too

Reply:
good catch, fixed.

---
In reply to: [329280361](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r329280361) [](ancestors = 329280361)